### PR TITLE
BaSP-TG-01: super admin table

### PR DESCRIPTION
**Pull request by kolodani**
functions: show, create, edit and delete

### CODE REVIEW

File: src/Components/SuperAdmins/index.js [hunk 1]
```diff
@@ -1,9 +1,106 @@
+import { useState, useEffect } from 'react';
 import styles from './super-admins.module.css';
+import SuperAdminsTable from './Table';
+import SuperAdminsEdit from './Edit';
+import SuperAdminsCreate from './Create';
+//import Modal from './Modal';
```
Comment (FacuDLF) [inline src/Components/SuperAdmins/index.js:6]: It should be erase the line commented.

**Review by FacuDLF — DISMISSED** [review]

File: src/Components/SuperAdmins/Edit/index.js [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+            setModalTitle('Success');
+            setModalMessage(res.message.toString());
+          } else {
+            setModalTitle('Error');
+            setModalMessage(res.message[0].message);
+          }
+        });
+    } catch (error) {
+      console.log('error', error.message);
+      console.error(error);
```
Comment (FacuDLF) [inline src/Components/SuperAdmins/Edit/index.js:46]: Leave only the catch errors with console.error. 
console.log() should be removed

**Review by FacuDLF — CHANGES_REQUESTED** [review]
Using try/catch, avoid to use console.log to catch errors. Should be catched using console.error() insted.

**Review by FacuDLF — APPROVED** [review]

**Review by anapaulib — APPROVED** [review]
Done

**Pull request merged**